public/i18n.js

```diff
@@ -11,10 +11,45 @@ const locales = {
   "es-ES": "i18n/es-es.json"
 };
 
-$(function () {
+const getConfigForLocale = (locale) => {
+  if (!locale) {
+    return null;
+  }
+
+  if (locales[locale]) {
+    return {
+      locale: locale,
+      map: {
+        [locale]: locales[locale]
+      }
+    };
```

Comment:
This might be weird, but the return could just be:

```javascript
return {
  locale: locale,
  [locale]: locales[locale]
};
```

Then you can pass the returned localeConfig directly to both $.i18n() and load()